frameworks/digital-outcomes-and-specialists/questions/brief-responses/essentialRequirements.yml

```diff
@@ -1,4 +1,5 @@
 name: Give evidence of the essential skills and experience
+short_name: Essential skills and experience
```

Comment:
With this one can't the `name` be `Essential skills and experience` and keep the long version for `question`?